### PR TITLE
Updated dependencies, added new options and validation

### PR DESCRIPTION
**Pull request by FrogTheFrog**
Added new option for a "raw" string. When using "raw-loader", a returned string is stringified - wrapped in quotes, which is can be an undesired behavior. New option will check if it is already a string and will not stringify it.

In addition to new option I updated dependencies, readme and webpack configurations. I also added JSON-scheme validator to prevent incorrect loader options.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,3 +55,47 @@ With each internal resource requested through require rather than just a plain p
...
 }
 ```
+
+## Options
+
+Additional options may be applied to loader. If incorrect is specified, loader will throw an error.
+
+|Option|Type|Default value|Description|
+|---|---|---:|---|
+|`rawString`|**_Boolean_**|false|Ensures that no extra brackets are added if the type of returned value is a **_string_** |
```
Comment (Swiftwork) [inline README.md]: > Ensures that no extra **brackets** are added if the type of returned value is a **_string_**

Should be this?

> Ensures that no extra **quotes** are added if the type of returned value is a **_string_**

**Review by Swiftwork — CHANGES_REQUESTED** [review]
Everything looks good, but please check the reviewed quote and add "package-lock.json" to the .gitignore. If it looks good after that I'll merge.

File: README.md (hunk 1 above)
Comment (FrogTheFrog) [inline README.md]: Done.

**Comment by FrogTheFrog** [comment]
@Swiftwork Done.

**Comment by Swiftwork** [comment]
Thank you @FrogTheFrog, good work!

**Pull request merged**